.drone.starlark

```diff
@@ -566,7 +566,8 @@ def buildDockerImage():
 			'DOCKER_HOST': 'tcp://docker:2375'
 		},
 		'commands': [
-			'docker build -t owncloud/phoenix:${DRONE_COMMIT_SHA}-${DRONE_BUILD_NUMBER} .'
+			'docker build -t owncloud/phoenix:${DRONE_COMMIT_SHA}-${DRONE_BUILD_NUMBER} .',
+			'docker push owncloud/phoenix:latest'
```

Comment:
You need to add `docker tag owncloud/phoenix:${DRONE_COMMIT_SHA}-${DRONE_BUILD_NUMBER} owncloud/phoenix:latest`

Reply:
done